### PR TITLE
R4 - Update AllergyIntolerance Overview

### PR DESCRIPTION
**Pull request by learclite**
Description
----
Add supported references to AllergyIntolerance recorder, asserter and note.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by learclite** [comment]
Before : 
Screenshot 2022-09-01 at 15 37 41

After commit [f40bd40](https://github.com/cerner/fhir.cerner.com/pull/885/commits/f40bd404ce47f91ba7cf15dd3f3021299b35b5ab)
Screenshot 2022-09-01 at 15 36 16

**Review by lstefan4 — APPROVED** [review]

File: content/millennium/r4/clinical/summary/allergy-intolerance.md [hunk 1]
```diff
@@ -19,24 +19,31 @@ If the consumer is sending in what is deemed to be a duplicate (codified or free
...
+* [Date of allergy onset](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.onset){:target="_blank"}
+* [Date/Time recorded](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recordedDate){:target="_blank"}
+* [Who recorded](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recorder){:target="_blank"}
+  * [Reference](https://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Practitioner](https://hl7.org/fhir/r4/practitioner.html){:target="_blank"})
+* [Who reported](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.asserter){:target="_blank"}
+  * [Reference](https://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Patient](https://hl7.org/fhir/R4/patient.html){:target="_blank"})
+* [Comment](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.note){:target="_blank"}
+  * [Reference](https://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Practitioner](https://hl7.org/fhir/r4/practitioner.html){:target="_blank"})
```
Comment (cerbs) [inline content/millennium/r4/clinical/summary/allergy-intolerance.md:44]: Comment is an Annotation. The comment's author is a Reference

File: content/millennium/r4/clinical/summary/allergy-intolerance.md [hunk 2]
```diff
@@ -19,24 +19,31 @@ If the consumer is sending in what is deemed to be a duplicate (codified or free
...
+* [Patient with allergy/intolerance](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.patient){:target="_blank"}
+* [Encounter](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.encounter){:target="_blank"}
+* [Date of allergy onset](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.onset){:target="_blank"}
+* [Date/Time recorded](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recordedDate){:target="_blank"}
+* [Who recorded](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recorder){:target="_blank"}
+  * [Reference](https://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Practitioner](https://hl7.org/fhir/r4/practitioner.html){:target="_blank"})
+* [Who reported](https://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.asserter){:target="_blank"}
+  * [Reference](https://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Patient](https://hl7.org/fhir/R4/patient.html){:target="_blank"})
```
Comment (cerbs) [inline content/millennium/r4/clinical/summary/allergy-intolerance.md:42]: Add another bullet point to indicate we support contained References to PractitionerRole and RelatedPerson

**Review by cerbs — CHANGES_REQUESTED** [review]

File: content/millennium/r4/clinical/summary/allergy-intolerance.md (hunk 2 above)
Comment (learclite) [reply content/millennium/r4/clinical/summary/allergy-intolerance.md:42]: Changed in commit b1c06ec :
Screenshot 2022-09-07 at 14 14 40

Let me know if the wording is alright.

**Review by cerbs — APPROVED** [review]

**Review by davischristi — APPROVED** [review]

**Pull request merged**